### PR TITLE
Fix:revolving dot issue #140

### PR DESCRIPTION
**Pull request by Atishay-J**
## This fixes Revolving dot issue #140 

Both the circles are now scaling up and down properly without getting cropped.

revolving fix

### Some additions 

- Added support for secondary color
- Added support to change stroke width

### CODE REVIEW

**Comment by Atishay-J** [comment]
I am not sure about what package version should I give it.
while this change is backward compatible, this also makes the circle radius comparatively smaller than the current version.
users might need to change the radius to a bigger number.

**Comment by mhnpd** [comment]
@Atishay-J  Let's update the props type of  `Radius` and the `Stroke Width` for this spinner to number? I think, we can not just support any number range for such value. If you have played around with configuration let's limit it with the type check. `i.e: radius: 4 | 5 | 6 | 7 | 8 |`. At least user will get a console warn if they tries to pass radius value outside of desired range. I think this is an issue with other spinner as well. 

Please feel free  to open a PR for other spinner if you like.

File: src/loader/RevolvingDot.tsx [hunk 1]
```diff
@@ -5,13 +5,14 @@ import { BaseProps, DEFAULT_COLOR, DEFAULT_WAI_ARIA_ATTRIBUTE } from '../type'
 interface RevolvingDotProps extends BaseProps {
   radius?: string | number
   secondaryColor?: string
+  strokeWidth?: string | number
```
Comment (mhnpd) [inline src/loader/RevolvingDot.tsx]: Rethink about the props type for the spinner. Number will make more sense and we can limit it to supported range.

**Comment by mhnpd** [comment]
> I am not sure about what package version should I give it. while this change is backward compatible, this also makes the circle radius comparatively smaller than the current version. users might need to change the radius to a bigger number.

I think it should be a major update, existing spinner would change its looks and feel silently.

File: src/loader/RevolvingDot.tsx (hunk 1 above)
Comment (Atishay-J) [reply src/loader/RevolvingDot.tsx]: I agree, we can keep it a number.
But I think we should not have a limit to it, people may want to have granular control for the component.

In this case they can mix and match `radius` and `strokeWidth` according to their preference

**Pull request merged**